node/8/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:8
+
+ENV PHANTOMJS_VERSION 2.1.1
```

Comment:
Same question as in https://github.com/busbar-io/busbar-base-images/pull/6#discussion_r341437087

Reply:
Left comment there because Leon has personal opinion about this also.

Reply:
We decided to keep PhantomJS for v8.